k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-io-svc.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.services.create -}}
 {{- $svc := .Values.services.io -}}
 {{- if gt $svc.count 0 -}}
```

Comment:
Could you make this fix too? (see #206)

```suggestion
{{- if gt (int $svc.count) 0 -}}
```

Reply:
Yes, certainly!  I'll make sure that's in the net update.

Reply:
This has been addressed.